server/handler/status.go

```diff
@@ -39,12 +39,17 @@ func (h *Status) Handle(ctx context.Context, eventType, deliveryID string, paylo
 	}
 
 	repo := event.GetRepo()
+	checkName := event.GetName()
+	checkState := event.GetState()
+
 	owner := repo.GetOwner().GetLogin()
 	repoName := repo.GetName()
 	installationID := githubapp.GetInstallationIDFromEvent(&event)
 	ctx, logger := githubapp.PrepareRepoContext(ctx, installationID, repo)
 
-	if event.GetState() != "success" {
+	logger.Debug().Msgf("Received status event for %s with state %s", checkName, checkState)
```

Comment:
Since you're adding/changing these events, could we make sure they all use a consistent message, i.e. same capitalization, roughly the same text, etc?

Reply:
Capitalization fixed. Changed this message to include the state where other messages include the action. e.g. "Received status success event from policy-bot: develop"